docs/cli.md

```diff
@@ -204,14 +212,46 @@ $ [object Promise]
 $ >> exit
 ```
 
-::: tip
+:::
 
-Remember, the `-h` or `--help` flag can always be used after any command or subcommand to receive more detailed information about its usage. This guide is a brief overview, and the `help` command will provide the most current and detailed instructions.
+## Logs
 
-:::
+Explore logs from deployed smart functions. The full output of the log can also be checked from the logs/kernel.log file.
 
-# Logs
+### Usage:
+
+```bash
+jstz logs trace [OPTIONS] <ALIAS|ADDRESS>
+```
 
-In order to see the output logs of the above commands, you should check the logs/kernel.log file.
-We provide `scripts/commands/view-console.sh` script through which conveniently shows the tail of most relevant parts of the log file.
-Alternatively you can use tools like `tail` or `grep` for lookup in the logs.
+### Arguments:
+
+- `<ALIAS|ADDRESS>`: The function's address or alias.
+
+### Options:
+
+- `--level (-l) <level>`: Specifies the level of log. Default is `log`.
+
+- `--network (-n) <NETWORK>`: Specifies the network from the config file. Use `dev` for the local sandbox.
+
+### Example
+
+```bash
+$ export logging=tz1VEJLogF4PArzDq6pvWDhw8WjTMgCwsB93 # Address of the previously deployed smart function examples/logs.js
+$ cargo run -- logs trace "${logging}"
```

Comment:
Is it possible to have an example with a function alias? I would rather avoid asking to do the exports as they are not needed anymore with aliases.